docs/pages/access-controls/reference.mdx

```diff
@@ -373,3 +374,188 @@ Here is an explanation of the fields used in the `where` and `filter` conditions
 
 Check out our [predicate language](../reference/predicate-language.mdx#scoping-allowdeny-rules-in-role-resources)
 guide for a more in depth explanation of the language.
+
+## Template variables and functions
+
+Teleport provides several template variables and functions to enable more robust access control:
```

Comment:
nit: not sure robust is the word here, maybe flexible, dynamic, powerful. Can we link to the dedicated template variables page as an overview here? And backlink from that page to here for a complete reference